metrics/prometheus_test.go

```diff
@@ -10,9 +10,20 @@ import (
 	th "github.com/containous/traefik/testhelpers"
 	"github.com/containous/traefik/types"
 	"github.com/prometheus/client_golang/prometheus"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
```

Comment:
This may be a dumb question, but why import this again, with a different name?

Reply:
Ahah lol, auto

Reply:
I will fix it 😝

Reply:
… and fixed :stuck_out_tongue: